Gordon360/Models/CCT/SeriesType.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT
+{
+    [Table("SeriesType", Schema = "RecIM")]
+    public partial class SeriesType
+    {
+        public SeriesType()
+        {
+            Series = new HashSet<Series>();
+        }
+
+        [Key]
+        public int ID { get; set; }
+        [Required]
+        [StringLength(256)]
+        [Unicode(false)]
+        public string Description { get; set; }
+        [Required]
+        [StringLength(4)]
+        [Unicode(false)]
+        public string TypeCode { get; set; }
```

Comment:
It's a code for our series type: i.e. RR = round robin, SE = single elim. We had this discussion a while ago when I originally wanted to implement an ID + unique code identifier (like Jenzabar) which we eventually found to be redundant. However, I found it useful for readability specifically in this use case as I much prefer using a type code when handling Auto-Scheduling since it would feel arbitrary to have the autoscheduling switch statement triggered by a number.